service/http.proxy/src/main/java/org/kaazing/gateway/service/http/proxy/HttpProxyServiceHandler.java

```diff
@@ -132,10 +152,11 @@ private URI computeConnectPath(URI connectURI) {
         @Override
         public void operationComplete(ConnectFuture future) {
             if (future.isConnected()) {
-                DefaultHttpSession connectSession = (DefaultHttpSession)future.getSession();
+                DefaultHttpSession connectSession = (DefaultHttpSession) future.getSession();
 
                 if (LOGGER.isTraceEnabled()) {
-                    LOGGER.trace("Connected to " + getConnectURIs().iterator().next() + " ["+acceptSession+"->"+connectSession+"]");
+                    LOGGER.trace("Connected to " + getConnectURIs().iterator().next() + " [" + acceptSession + "->"
```

Comment:
Now that you are changing this line for formatting, s/getConnectURIs().iterator().next()/connectURI. same goes below.
Then I suppose you could put that in one line